air/src/trace/main_trace.rs

```diff
@@ -43,6 +54,13 @@ impl<'a> MainTrace<'a> {
         self.columns.num_rows()
     }
 
+    pub fn get_column_range(&self, range: Range<usize>) -> Vec<Vec<Felt>> {
+        range.fold(vec![], |mut acc, col_idx| {
+            acc.push(self.get_column(col_idx).to_vec());
+            acc
+        })
+    }
```

Comment:
This could be a very expensive method as it clones trace columns. However, as far as I can tell, this is used only for testing purposes - so, this should be fine. However, ideally, we'd put this method behind a feature.

Currently, we run tests under  the `internals` feature (which we should probably rename to `testing` - but that's for another PR). Not sure how straight-forward would it be to add this feature here as well. If it is not straight-forward, we can do it in a future PR and just leave a `TODO` here for now to make sure it is clear that this method is for testing purposes only.

Reply:
put the get_column_range function behind the internals feature.